repos/system_upgrade/el7toel8/actors/quaggadaemons/libraries/quaggadaemons.py

```diff
@@ -0,0 +1,37 @@
+from leapp.libraries.stdlib import CalledProcessError, api, run
+from leapp.models import QuaggaToFrrFacts
+
+QUAGGA_DAEMONS = [
+    'babeld',
+    'bgpd',
+    'isisd',
+    'ospf6d',
+    'ospfd',
+    'ripd',
+    'ripngd',
+    'zebra'
+]
+
+
+def _check_service(name, state):
+    try:
+        run(['systemctl', 'is-{}'.format(state), name])
+        api.current_logger().debug('{} is {}'.format(name, state))
```

Comment:
this is not the proper way of variable strings in logging. See the comments bellow